### DISCUSSION TITLE
sir g markdown ma file ko jb pdf karty hain tb bar bar error a jata hai . is ki kia reason .please help me

### DISCUSSION THREAD

**Discussion by waqarahmad45** [AammarTufail/python_ka_chilla2023#23] [Q&A]
Assalam o Alaikum 
sir g markdown ma file co jib pdf Karty Hain tb bar error a jota hai . is ki kya reason  hai .please help me.

**Comment by AammarTufail** [comment]
> ```
> Assalam o Alaikum 
> ```
> 
> sir g markdown ma file co jib pdf Karty Hain tb bar error a jota hai . is ki kya reason hai .please help me.

This is your assignment and you should do it.

**Reply by AzamBukhari** [reply]
Yeah bhi theek hai.. :)
@waqarahmad45 We have another for the same issue. We all are working on it. Join it may be APKO KUCH MUDED mil jaey..

Lugey raho munna bhai

**Reply by waqarahmad45** [reply]
Assalam o Aalikum 
bhai g ap  3 point pr jaye .pr export pr html pr click krein.then apko window milay gi 
tb ap ctrol +p press krein.is k baad ap save in pdf pr click krein.
please reply me .
